packages/react-components/src/stories/Introduction.mdx

```diff
@@ -15,9 +15,14 @@ export const link =
 
 <Meta title="Introduction" />
 
-# B.C. Design System React Components
+# B.C. Design System React Components v{packageJson.version}
```

Comment:
This new chunk **should** always work but let's write it like this instead to make sure:

```js
{packageJson?.version ? `v${packageJson.version}` : null}
```